src/lib/data/releases.json

```diff
@@ -106,5 +106,41 @@
                 "url": "https://github.com/oras-project/oras/releases/download/v1.1.0/oras_1.1.0_windows_amd64.zip"
             }
         }
+    },
+    "1.2.0": {
```

Comment:
Since ORAS v1.2.0 are also released to FreeBSD and Linux ppc64le, riscv64 and s390x. Please 

1) Add those new releases
2) Add corresponding parsing logic from node OS (see possible [platform ](https://nodejs.org/api/os.html#osplatform)and [arch](https://nodejs.org/api/os.html#osarch)) to supported release of ORAS.


https://github.com/marketplace/actions/cross-platform-action#supported-platforms